src/transformers/image_utils.py

```diff
@@ -320,7 +320,7 @@ def load_image(image: Union[str, "PIL.Image.Image"], timeout: Optional[float] =
 
             # Try to load as base64
             try:
-                b64 = base64.b64decode(image, validate=True)
```

Comment:
Could you explain a bit why `b64decode` here can't handle the data encoded by `encodebytes`?